crates/paralegal-policy/src/context.rs

```diff
@@ -321,7 +351,7 @@ impl Context {
             })
     }
 
-    /// Return an example pair for a data flow from an source from `from` to a sink
+    /// Return an example pair for a flow from an source from `from` to a sink
     /// in `to` if any exist.
     pub fn any_data_flows<'a>(
```

Comment:
I think it would make sense, if we're generifying `flows_to` to also generify this to `any_flows_to` with the same optional controller id and and edge type argument.

Reply:
done